server/sonar-server/src/main/java/org/sonar/server/batch/UsersAction.java

```diff
@@ -78,8 +79,10 @@ public void handle(Request request, Response response) throws Exception {
     }
   }
 
-  private void handleUser(UserDoc user, BatchInput.User.Builder userBuilder, OutputStream out) {
-    userBuilder.setLogin(user.login())
+  private static void handleUser(UserDoc user, BatchInput.User.Builder userBuilder, OutputStream out) {
+    userBuilder
+      .clear()
```

Comment:
to be bullet-proof if we don't update all the fields each time. It's the same builder reference which is passed in a loop.